lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -43,6 +43,15 @@ using llvm::cast;
 using llvm::dyn_cast;
 using llvm::isa;
 
+namespace {
+llvm::cl::OptionCategory bundleSaverCat("Bundle Options");
```

Comment:
@mciprian13  Could you move these definitions into `LLVMIRCodeGen/CommandLine.cpp`? We have `bundleSaverCat` and one of the bundle options defined there already. Let's keep it in one place.

Reply:
Sorry ... it should have been obvious. I solved it.